### PR TITLE
perf(Tree): add components for benchmarking

### PR DESCRIPTION
**Pull request by silviuaavram**
Adds 2 perf examples. List with 60 list items vs Tree with 3 items that each have 20 List items.

### CODE REVIEW

File: docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      media={media}
+    />
+  )
+}
+
+const Tree60WithListItems = () => (
+  <Tree items={items} defaultActiveItemIds={['1', '2', '3']} renderItemTitle={titleRenderer} />
+)
+
+Tree60WithListItems.iterations = 5000
```
Comment (miroslavstastny) [inline docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx]: This controls flamegrill perf tests (see docs here: https://github.com/microsoft/fluent-ui-react/blob/master/.github/test-a-feature.md#performance-tests).
This perf example probably takes hundreds of seconds to render - it is enough to run just one iteration for flamegrill tests.
You can run `yarn perf:test` to verify.

File: docs/src/examples/components/List/Performance/ListWith60ListItems.perf.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    key: `key${i}`,
+    content: `${contents[i % contents.length]}`,
+    media: <Image src={`${avatars[i % avatars.length]}`} avatar />,
+    headerMedia: `${headerMedias[i % headerMedias.length]}`,
+    header: `${headers[i % headers.length]}`,
+  }))
+
+const ListCommonPerf = () => <List items={itemData()} />
+
+ListCommonPerf.iterations = 100
```
Comment (miroslavstastny) [inline docs/src/examples/components/List/Performance/ListWith60ListItems.perf.tsx]:
```suggestion
ListCommonPerf.iterations = 1
```

File: docs/src/examples/components/List/Performance/ListWith60ListItems.perf.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  _.times(60, i => ({
+    id: `test${i}`,
+    key: `key${i}`,
+    content: `${contents[i % contents.length]}`,
+    media: <Image src={`${avatars[i % avatars.length]}`} avatar />,
+    headerMedia: `${headerMedias[i % headerMedias.length]}`,
+    header: `${headers[i % headers.length]}`,
+  }))
+
+const ListCommonPerf = () => <List items={itemData()} />
```
Comment (mnajdova) [inline docs/src/examples/components/List/Performance/ListWith60ListItems.perf.tsx]: Is it intentional that this is a function? Why don't we just have items as list inside? We can change it for const

File: docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+  {
+    id: '3',
+    title: 'House Stark',
+    items: itemData('list-3'),
+  },
+]
+
+const titleRenderer = (Component, { content, header, headerMedia, media, ...restProps }) => {
+  return !header ? (
```
Comment (layershifter) [inline docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx:56]: Let's add a comment there what it does

**Review by layershifter — APPROVED** [review]

File: docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
+import { Tree, Image, List } from '@fluentui/react'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+const avatars = [
+  'public/images/avatar/small/matt.jpg',
```
Comment (miroslavstastny) [inline docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx]: Do not do any additional network request. 
Do this: https://github.com/microsoft/fluent-ui-react/blob/880790637ed2113fbb500b28f8c1126be5347061/docs/src/examples/components/Chat/Performance/ChatWithPopover.perf.tsx#L6-L14

**Review by miroslavstastny — CHANGES_REQUESTED** [review]

File: docs/src/examples/components/List/Performance/ListWith60ListItems.perf.tsx (hunk 3 above)
Comment (silviuaavram) [reply docs/src/examples/components/List/Performance/ListWith60ListItems.perf.tsx]: No, I think it was a copy paste from Tree perf prototype. Will change it. Thanks!

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**